### PR TITLE
Update index.js.flow

### PR DESCRIPTION
**Pull request by wolverineks**

### CODE REVIEW

**Comment by wolverineks** [comment]
Not sure if this file is created automatically, but it looks like the new core plugin functions should be here to make flow happy

File: index.js.flow [hunk 1]
```diff
@@ -12,6 +12,8 @@ import {
 
 export * from './src/types/types.js'
 
+declare export function addEdgeCorePlugins(): mixed
```
Comment (swansontec) [inline index.js.flow]: Good catch, but we need to set up the parameter type here.

**Comment by swansontec** [comment]
Cherry-picked, adjusted, & merged.

**Comment by swansontec** [comment]
Published in v0.15.4.

**Pull request closed without merging**